kafka-rest/src/main/java/io/confluent/kafkarest/controllers/TopicManagerImpl.java

```diff
@@ -130,8 +132,15 @@
             });
   }
 
-  private CompletableFuture<List<Topic>> describeTopics(String clusterId, List<String> topicNames) {
-    return KafkaFutures.toCompletableFuture(adminClient.describeTopics(topicNames).all())
+  private CompletableFuture<List<Topic>> describeTopics(
+      String clusterId, List<String> topicNames) {
+    return KafkaFutures.toCompletableFuture(
+            adminClient
+                .describeTopics(
+                    topicNames,
+                    new DescribeTopicsOptions()
+                        .includeAuthorizedOperations(true))
```

Comment:
We should only set it if user is asking for it. I assume it is optional because there's a cost involved in asking for it :). That means you should pipe include_authorized_operations from the resource to the manager.